### ISSUE TITLE
Data migration to purge instances with `deleted_at` set

### ISSUE THREAD

**Issue by jnm** [kobotoolbox/kobocat#733]
Once #732 is merged and `master` is merged into `beta` (or vice versa), we need to add a new data migration for self-hosters that automatically removes any old submissions with `deleted_at` (`_deleted_at` in Mongo) from the databases. Basically, take the code from https://github.com/kobotoolbox/kobocat/issues/696#issuecomment-809622367 and plop it into a data migration.
